pkgconfig/README.md

```diff
@@ -38,15 +38,15 @@ commands.
 ```
 cflags=`pkg-config oeenclave-clang --cflags`
 libs=`pkg-config oeenclave-clang --libs`
-$ clang-10 ${cflags} -o enc enc.c ${libs}
+$ clang-11 ${cflags} -o enc enc.c ${libs}
```

Comment:
@CyanDevs  - As an example, we only support clang-11 here

Reply:
I will add clarification to this README that clang-10 is also supported, despite the example showing clang-11.

Reply:
I suggest we say something like clang-11 is preferred, but clang-10 is supported